appAndroid/post/src/main/java/com/gchristov/newsfeed/post/PostActivity.kt

```diff
@@ -190,7 +192,7 @@ private fun PostAuthor(author: String) {
                 style = Theme.typography.subtitle,
             )
             AppText(
-                text = stringResource(R.string.post_read_time),
+                text = stringResource(R.string.post_read_time).replace("1", readingTimeMinutes.toString()),
```

Comment:
Lets use [positional formatting](https://developer.android.com/jetpack/compose/resources#strings) here, rather than replacing the text.

Reply:
Applied in next commit